Move NFT and fractional-NFT to archive folder with warnings (#712)
FuelLabs/sway-applications: archived_apps/README.md

## Archived Apps

> The following applications have been temporarily archived as they are awaiting the finalised release of of the [Sway multi-token standard](https://github.com/FuelLabs/rfcs/pull/17) before being updated. In their current form, they do not comply with this standard and as such should be considered outdated.

Reviewer: ```suggestion
The following applications have been temporarily archived as they are awaiting the release of the [Sway multi-token standard](https://github.com/FuelLabs/rfcs/pull/17) before being updated. In their current form, they do not comply with this standard and as such should be considered outdated.
```

> **Warning**

> > It is not advised to use them for production purposes, use at your own risk

Reviewer: ```suggestion
> It is inadvisable to use them for production purposes.
```

- fractional-NFT

> - NFT

Reviewer: Either link to them or remove this

Author: Linked
